packages/ui-client/src/components/PasswordVerifier/PasswordVerifierItem.tsx

```diff
@@ -1,43 +1,45 @@
 import { Box, Icon } from '@rocket.chat/fuselage';
-import { AllHTMLAttributes, ComponentProps } from 'react';
+import type { PasswordPolicyValidation } from '@rocket.chat/ui-contexts';
+import { useId } from 'react';
+import { useTranslation } from 'react-i18next';
 
-const variants: {
-	[key: string]: {
-		icon: ComponentProps<typeof Icon>['name'];
-		color: string;
-	};
-} = {
+const variants = {
 	success: {
-		icon: 'success-circle',
+		name: 'success-circle',
+		label: 'Success',
 		color: 'status-font-on-success',
 	},
 	error: {
-		icon: 'error-circle',
+		name: 'error-circle',
+		label: 'Error',
```

Comment:
WDYT instead of having this object with "loose" translation keys, make a function that receives `valid: boolean` and `t: TranslationFn` and returns the translated text instead of the key?

I'd rather have the translation key in the fn's invocation instead of as a property, specially because it makes it easier to debug and read.